changelog/0089-remove-parser-class.yml

```diff
@@ -0,0 +1,24 @@
+title: Remove PARSER_CLASS from redis settings
+key: remove-parser-class
+date: 2025-07-10
+optional_per_env: no
+# (optional) Min version of HQ that MUST be deployed before this change can be rolled out (commit hash)
+min_commcare_version:
+# (optional) Max version of HQ that can be deployed before this change MUST be rolled out (commit hash)
+max_commcare_version:
```

Comment:
Could this be set to the branch (HEAD commit) that upgrades redis?

Reply:
oh yeah good point